### PR TITLE
Update base_rate and exchange_rate by renaming with get_ prefix

### PR DESCRIPTION
**Pull request by j450h1**

### CODE REVIEW

File: R/base_rate.R [hunk 1]
```diff
@@ -1,39 +1,45 @@
-#' Base Rate
```
Comment (philip-khor) [inline R/base_rate.R:1]: why is git diff chucking out the entire bit :(

Comment (j450h1) [reply R/base_rate.R:1]: Not sure..

**Review by j450h1 — COMMENTED** [review]
Looks good to me

**Comment by philip-khor** [comment]
> 
> 
> Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?

I think we're good to go, the checks should pass now if I've done everything correctly. 

Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.

File: bnmr.Rproj [hunk 2]
```diff
@@ -1,20 +1,20 @@
-Version: 1.0
```
Comment (philip-khor) [inline bnmr.Rproj:1]: there's no difference here :( why git why

**Review by philip-khor — APPROVED** [review]
I'm happy with this PR, but will leave this open for a while to see if I can fix the large diffs. Thanks!

**Comment by j450h1** [comment]
> > Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?
> 
> I think we're good to go, the checks should pass now if I've done everything correctly.
> 
> Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.



> > Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?
> 
> I think we're good to go, the checks should pass now if I've done everything correctly.
> 
> Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.



> > Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?
> 
> I think we're good to go, the checks should pass now if I've done everything correctly.
> 
> Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.

Got it. Makes sense. 

Well, I'm on macOS, maybe I can print sessionInfo() if you think that would help? That's the only thing I can think of right now - am I right to guess you are on Windows?

**Comment by philip-khor** [comment]
> > > Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?
> > 
> > 
> > I think we're good to go, the checks should pass now if I've done everything correctly.
> > Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.
> 
> > > Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?
> > 
> > 
> > I think we're good to go, the checks should pass now if I've done everything correctly.
> > Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.
> 
> > > Interesting never seen that function before. Do you need me to update anything else before accepting the pull request?
> > 
> > 
> > I think we're good to go, the checks should pass now if I've done everything correctly.
> > Would appreciate some help looking into why we're getting the huge diffs here though. Got the same issue with the last PR I accepted, the problem with huge diffs is it's going to be quite hard to identify the changes in each commit.
> 
> Got it. Makes sense.
> 
> Well, I'm on macOS, maybe I can print sessionInfo() if you think that would help? That's the only thing I can think of right now - am I right to guess you are on Windows?

Yep that's right, that could definitely be it. Discarding changes for the Rproj could help, but not sure how to fix base_rate.

**Comment by philip-khor** [comment]
thanks!

**Pull request merged**